aloscene/points_2d.py

```diff
@@ -441,7 +441,7 @@ def _rotate(self, angle, **kwargs):
         points : aloscene.Point2d
             rotated points
         """
-        points = self.clone()
+        points = self.xy().clone()
```

Comment:
clone is redundant , xy() is already cloning the vector.

Reply:
done